rule/package-comments.go

```diff
@@ -31,9 +31,48 @@ func (*PackageCommentsRule) Apply(file *lint.File, _ lint.Arguments) []lint.Fail
 	fileAst := file.AST
 	w := &lintPackageComments{fileAst, file, onFailure}
 	ast.Walk(w, fileAst)
+	failures = append(failures, checkPackageComment(file.Pkg)...)
 	return failures
 }
 
+var checkPackageCommentCache = map[*lint.Package]struct{}{}
```

Comment:
warn: the same rule is concurrently applied to multiple files, then accessing this map might produce race conditions

Reply:
I though there is not concurrency.
replaced with sync.Map